src/main/java/nurseybook/logic/commands/EditCommand.java

```diff
@@ -94,7 +94,7 @@ public CommandResult execute(Model model) throws CommandException {
         Elderly elderlyToEdit = lastShownList.get(index.getZeroBased());
         Elderly editedElderly = createEditedElderly(elderlyToEdit, editElderlyDescriptor);
 
-        if (elderlyToEdit.isSameElderly(editedElderly)) {
+        if (elderlyToEdit.equals(editedElderly)) {
```

Comment:
Could I check what is the rationale for this change?

Reply:
It's to improve error handling, if user enters the same field during editElderly, it will prompt an error saying "Fields entered are the same as before".